src/HW2/intSwapWithVariable.java

```diff
@@ -1,5 +1,5 @@
 package HW2;
-
+//задача 2
 public class intSwapWithVariable {
```

Comment:
Class names should start with the upper case letters. For example: IntSwapWithVariable